securedrop/i18n.py

```diff
@@ -137,12 +140,20 @@ def validate_locale_configuration(config: SecureDropConfig, babel: Babel) -> Set
     available = set(babel.list_translations())
     available.add(Locale.parse(FALLBACK_LOCALE))
 
+    # These locales are supported in the current version of securedrop-app-code.
+    try:
+        with open(I18N_CONF) as i18n_conf_file:
+            i18n_conf = json.load(i18n_conf_file)
+        supported = parse_locale_set(i18n_conf["supported_locales"].keys())
+    except FileNotFoundError:
```

Comment:
In what case would the file not be found? If i18n.json is missing, that feels like a fatal error to me.

Reply:
Clarified in bdffeede308380554c91cd1cb7f6822f10ba346c.  Let me know if this doesn't address your concern.

Reply:
Sorry, but why would it not be available under test? Is something deleting it?

Reply:
https://github.com/freedomofpress/securedrop/commit/2b1214c13269f03660b7a6714be61a46bbff3acb has a green check. I'll turn it into a PR, but we don't need to get it into 2.12.

If I had to guess, if it was failing earlier possibly because the path was relative, but now that you changed it to an absolute path, it works?